src/store/store-items.js

```diff
@@ -50,15 +51,20 @@ const mutations = {
   },
   setListName(state, value) {
     state.listName = value
+  },
+  setListId(state, value) {
+    state.listId = value
   }
 }
 
 const actions = {
   updateItem({ dispatch }, payload) {
     dispatch('fbUpdateItem', payload)
+    dispatch('fbUpdateListMetaData')
   },
   deleteItem({ dispatch }, payload) {
     dispatch('fbDeleteItem', payload)
+    dispatch('fbUpdateListMetaData')
```

Comment:
This will update the lists store with the meta data (totals).
It might be need to verify how many times it gets call.
Ideally every time user exits from the list the metadata should be updated.
Atm on update, add and delete it gets updated. There might be a better way to do it.